File: Day 22/Aayushi-Mittal.cpp
//To Find the maximum and minimum number in an array.

#include<iostream>
using namespace std;
int main()
{
  int n=10, arr[n], maximum=-100000, minimum=100000;
  cin>>n;
  for(int i=0; i<n; i++)
  {
    cin>>arr[i];
    if(maximum<arr[i])
        maximum = arr[i];
    if(minimum>arr[i])
        minimum = arr[i];
  }
  cout<<"> Maximum : "<<maximum<<endl;
  cout<<"> Minimum : "<<minimum<<endl;
}

File: Day 22/Aayushi-Mittal.py
# To Find the maximum and minimum number in an array.

n=int(input()) 
arr=[] 
maximum=-100000
minimum=100000
for i in range(0,n):
    arr.append(int(input()))
    if maximum<arr[i]:
        maximum = arr[i]
    if minimum>arr[i]:
        minimum = arr[i]
print("> Maximum : ",maximum)
print("> Minimum : ",minimum)
